rust_snuba/src/processors/spans.rs

```diff
@@ -6,17 +6,20 @@ use serde::{Deserialize, Serialize};
 use serde_json::Value;
 use uuid::Uuid;
 
-use crate::processors::utils::{default_retention_days, hex_to_u64, DEFAULT_RETENTION_DAYS};
+use crate::config::EnvConfig;
+use crate::processors::utils::{enforce_retention, hex_to_u64};
 use crate::types::{InsertBatch, KafkaMessageMetadata};
 
 pub fn process_message(
     payload: KafkaPayload,
     metadata: KafkaMessageMetadata,
+    config: &EnvConfig,
```

Comment:
can we wrap this in some kind of `MessageProcessorConfig` so that people start extending that struct instead of adding more and more arguments?

Reply:
yep, updated